test/HttpServer.test.ts

```diff
@@ -170,15 +168,8 @@ describe('HttpServer', () => {
                 fakeRequestExpress as Request,
                 fakeResponseExpress as Response
             );
-
-            assert.containsAllKeys(fakeResponseExpress.send?.args[0][0], [
-                'signedTx',
-                'transactionHash'
-            ]);
-            assert.deepEqual(
-                fakeResponseExpress.send?.args[0][0],
-                fakeResponseRelayTransaction
-            );
+            
+            fakeResponseExpress.send?.calledOnceWithExactly(fakeResponseRelayTransaction);
```

Comment:
Why the `send` property should be null if we define it at line 150?

Reply:
@kayseriiovlabs I've fixed this using the [`Pick` type](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys)